now/cli/parser.py

```diff
@@ -138,6 +138,39 @@ def set_logs_parser(sp):
     )
 
 
+def set_compare_parser(sp):
+    parser = sp.add_parser(
+        'compare',
+        help='Compare the performance of different flows.',
+        description='Compare the performance of different flows.',
+        formatter_class=_chf,
+    )
+
+    parser.add_argument(
+        '--path_semantic_scores',
+        help='Path to json file mapping flow IDs to a list of semantic scores configurations',
+        type=str,
+    )
+
+    parser.add_argument(
+        '--cluster',
```

Comment:
```suggestion
        '--clusters',
```
Intuitively has to be a list of clusters always, since we are comparing? Or could we also just use one cluster + different sem scores?

Reply:
Clusters is the right formulation. To specify semantic scores one needs to use the json file.